extensions/amp-pinterest/amp-pinterest.md

```diff
@@ -95,6 +95,10 @@ Contains the fully-qualified URL of the image intended to be pinned. If the pin
 
 Contains the default description that appears in the pin create form; please choose carefully, since many Pinners will close the form without pinning if it doesn't make sense.
 
+##### alt
+
+This property like the the `alt` attribute on an `<img>` tag specifies the alternate ext to be rendered in case the image specified can't be rendered.
```

Comment:
`s/ext/text`

I'd remove `to be rendered in case the image specified can't be rendered` and instead add ". If no `alt` is provided, it will be inferred from the PIN data provided by Pinterest API."

Reply:
Done